### ISSUE TITLE
Ctor should set default values for a property that returns an Enum

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#231]
To make it clear to the developer and to prevent accidentally changed behavior, properties that return `Enum` values should be set via the `.ctor`.

Following code **SHOULD** report an issue:
```c#
public class TestMe
{
    public TestMe() { }

    public StringComparison Comparison { get; }
}
```
Reason: The `.ctor` does not set the value directly, and there is also no value set via any property initializer.

In contrast to the example above, following code **should NOT** report an issue:
```c#
public class TestMe
{
    public TestMe() { }

    public StringComparison Comparison { get; } = StringComparison.OrdinalIgnoreCase;
}
```
Reason: The `.ctor` does not set the value directly, but it is set via property initializer.

**Comment by RalfKoban** [comment]
Following code should not report an issue:
```c#
public class TestMe
{
    public TestMe() { }

    public StringComparison Comparison { get; } = StringComparison.OrdinalIgnoreCase;
}
```
The ctor does not set the value directly, but it is set via property initializer.

**Comment by RalfKoban** [comment]
Maybe it is better to report the property itself and not the .ctor as issue. A codefix could then apply a property initializer.

**Comment by RalfKoban** [comment]
Resolved in 5ba0600f7059281cc8b3841c966ca19bdca9c396
